### DISCUSSION TITLE
Удалённый доступ к nfqws через ssh

### DISCUSSION THREAD

**Discussion by SVY76** [Anonym-tsk/nfqws-keenetic#111] [Q&A]
Привет.
Может, кто уже сделал. Как открыть доступ к ssh на 222 порту извне?
Я пробовал обойтись средствами своего кинетика, через маппинг портов - ничего не выходит, соединение не поднимается.

**Comment by Anonym-tsk** [comment]
Не нужно это делать.
Установите на кинетик впн и подключайтесь к нему снаружи.
